iphone/Classes/TiUIPicker.m

```diff
@@ -190,19 +191,6 @@ -(void)setType_:(id)type_
 	}
 }
 
-// We're order-dependent on type being set first, so we need to make sure that anything that relies
-// on whether or not this is a date picker needs to be set AFTER the initial configuration.
--(void)setSelectionIndicator_:(id)value
```

Comment:
Rather deleting the method , we should deprecate it by giving message to developers. What you think?